charts/gsp-cluster/values.yaml

```diff
@@ -47,6 +47,15 @@ githubAPIToken: ""
 #   requiredApprovalCount: 2
 #   scope: cluster
 
+cluster-autoscaler:
+  serviceMonitor:
+    enabled: true
+  nodeSelector:
```

Comment:
Not sure why we're targeting the `cluster-management` node group...that seems unnecessary.

Reply:
see above